frontend/src/index.pug

```diff
@@ -34,73 +34,73 @@ html
     script(src="https://cdn.jsdelivr.net/gh/highlightjs/cdn-release@9.14/build/languages/yaml.min.js")
   body
     #app
-      v-resizer(
-        v-if="userUpdated",
-        v-bind:is-tab-active="isTabActive",
-        v-on:close-tab="deactivateTab"
-      )
-        #summary-wrapper(slot="left")
-          .summary-status(v-if="loadedRepo<repoLength") Loading repos ... {{ loadedRepo }} / {{ repoLength }}
```

Comment:
Can move this line and the `loadedRepo` related attributes. We are not showing this anymore?

Reply:
do u mean remove this line?